lib/experimental/block-editor-settings.php

```diff
@@ -83,6 +83,7 @@ function gutenberg_get_block_editor_settings_experimental( $settings ) {
 		}
 	}
 
+	$settings['filters'] = WP_Duotone_Gutenberg::get_footer_assets();
```

Comment:
We shouldn't use this function, as it only outputs used filters, whereas in the post editor we need to load all available presets since users can add blocks to the post.

Reply:
I updated it to use the filter directly in duotone.php